taxi/urls.py

```diff
@@ -53,11 +53,9 @@
     ),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
```

Comment:
Ensure there are no duplicate URL patterns for the driver list and detail views. This can lead to conflicts in URL resolution.